classes/module/Module.php

```diff
@@ -1721,6 +1723,18 @@ public static function getModulesInstalled($position = 0)
         return Db::getInstance()->executeS($sql);
     }
 
+    /**
+     * Return active modules.
+     *
+     * @return array Modules
+     */
+    public static function getActiveModules(): array
```

Comment:
Method already available in:
```
- '@=service("prestashop.module_kernel.repository").getActiveModules()'
 ```

Reply:
But this is not usable in legacy context, which is why this is needed (for FO mainly)